airflow-core/src/airflow/ui/src/i18n/locales/de/admin.json

```
@@ -1,6 +1,161 @@
+  },
+  "connections": {
+    "add": "Verbindung hinzufügen",
+    "columns": {
+      "connectionId": "Verbindungs-ID",
+      "connectionType": "Verbindungstyp",
+      "host": "Hostname",
+      "port": "Port"
+    },
+    "connection_one": "Verbindung",
+    "connection_other": "Verbindungen",
+    "delete": {
+      "deleteConnection_one": "1 Verbindung löschen",
+      "deleteConnection_other": "{{count}} Verbindungen löschen",
+      "firstConfirmMessage_one": "Die folgende Verbindung wird gelöscht:",
+      "firstConfirmMessage_other": "Die folgenden Verbindungen werden gelöscht:",
+      "title": "Verbindung löschen"
+    },
+    "edit": "Verbindung bearbeiten",
+    "form": {
+      "connectionIdRequired": "Verbindungs-ID ist erforderlich",
+      "connectionIdRequirement": "Die Verbindungs-ID kann nicht nur aus Leerzeichen bestehen",
+      "connectionTypeRequired": "Die Auswahl eines Verbindungstyps ist erforderlich",
+      "extraFields": "Weitere Felder",
+      "extraFieldsJson": "JSON Ansicht der weiteren Felder",
+      "helperText": "Fehlt ein Verbindungstyp? Stellen Sie sicher, dass die notwendigen Provider-Pakete installiert sind.",
+      "selectConnectionType": "Verbindungstyp auswählen",
+      "standardFields": "Standardfelder"
+    },
+    "noRowMessage": "Keine Verbindungen gefunden",
+    "searchPlaceholder": "Verbindungen suchen",
     "test": "Verbindung testen",
-    "testDisabled": "Das Testen von Verbindungen ist deaktiviert. Der Administrator kann via Konfiguration das Testen freischalten."
+    "testDisabled": "Das Testen von Verbindungen ist deaktiviert. Der Administrator kann via Konfiguration das Testen freischalten.",
+    "typeMeta": {
+      "error": "Fehler beim Abrufen der Liste der Verbindungstypen",
+      "standardFields": {
+        "description": "Beschreibung",
+        "host": "Hostname",
+        "login": "Benutzername",
+        "password": "Passwort",
+        "port": "Port",
+        "url_schema": "Verbindungsschema"
+      }
+    }
+  },
+  "deleteActions": {
+    "button": "Löschen",
+    "modal": {
+      "confirmButton": "Ja, Löschen",
+      "secondConfirmMessage": "Diese Aktion kann nicht rückgängig gemacht werden. ",
+      "thirdConfirmMessage": "Sind Sie sicher?"
+    },
+    "selected": "Ausgewählt",
+    "tooltip": "Ausgewählte Verbindungen löschen"
+  },
+  "formActions": {
+    "reset": "Zurücksetzen",
+    "save": "Speichern"
+  },
+  "plugins": {
+    "columns": {
+      "source": "Quelle"
+    },
+    "importError_one": "Fehler beim Laden eines Plug-in",
+    "importError_other": "Fehler beim Laden von Plug-ins",
+    "searchPlaceholder": "Nach Datei suchen"
+  },
+  "pools": {
+    "add": "Pool hinzufügen",
+    "deferredSlotsIncluded": "Delegierte Tasks einschließen",
+    "delete": {
+      "title": "Pool löschen",
+      "warning": "Dies wird alle Metadaten zu diesem Pool entfernen. Damit kann es eine Auswirkung auf Tasks geben, die diesen Pool nutzen."
```

```suggestion
      "warning": "Dies wird alle Metadaten zu diesem Pool entfernen. Dies kann Tasks beeinträchtigen, die diesen Pool nutzen. Ein Task, der einem nicht existenten Pool zugewiesen ist, kann nicht geplant werden."
```

I think it might make sense if there is space to be explicit about what happens with tasks that don't have a pool.